go.mod

```diff
@@ -22,6 +22,94 @@ require (
 	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
 )
 
+require (
+	github.com/Microsoft/go-winio v0.4.16 // indirect
+	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 // indirect
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
+	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/acomagu/bufpipe v1.0.3 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
+	github.com/emirpasic/gods v1.12.0 // indirect
+	github.com/evanphx/json-patch v4.11.0+incompatible // indirect
```

Comment:
why are there 2 require blocks? is the second one generated by some script?

Reply:
>is the second one generated by some script?

By `go mod vendor` itself.

Yes, this will be the behaviour starting from go v1.17 & higher. If a module specifies go 1.17 or higher, the module graph includes only the immediate dependencies of other go 1.17 modules, not their full transitive dependencies.

Which is why, if you carefully observe the second section then almost all have `//indirect` labels.